subscription-engine/kafka-ws/app/Redis/RedisManager.js

```diff
@@ -37,10 +60,26 @@ class RedisManager {
      * When the client disconnects to redis the 'end' event is fired, there we can consider
      * the service is unhealthy
      */
-    this.redisClient.on('error', (error) => logger.warn(`${error}`));
-    this.redisClient.on('end', () => StateManager.signalNotReady(stateService));
+    this.redisClient.on('error', (error) => {
+      logger.error('Redis has an error:', error);
+      if (error.code === 'CONNECTION_BROKEN') {
```

Comment:
Shouldn't be better to put this snippet of code in a function? It is used in other places.

Reply:
You're right, I'll put it in Utils like I did with the health check